mesonbuild/modules/gnome.py

```diff
@@ -311,36 +311,6 @@ def _print_gdbus_warning() -> None:
                      mlog.bold('https://github.com/mesonbuild/meson/issues/1387'),
                      once=True)
 
-    def _get_dep(self, state: 'ModuleState', depname: str, native: bool = False,
-                 required: bool = True) -> Dependency:
-        kwargs = {'native': native, 'required': required}
-        # FIXME: Even if we fix the function, mypy still can't figure out what's
-        # going on here. And we really dont want to call interpreter
-        # implementations of meson functions anyway.
```

Comment:
We should probably preserve this comment when moving code around?

Reply:
Added back.